src/main/java/com/titanic/fork/web/controller/LoginController.java

```diff
@@ -0,0 +1,41 @@
+package com.titanic.fork.web.controller;
+
+import com.titanic.fork.service.account.LoginService;
+import com.titanic.fork.web.dto.request.account.NewPasswordRequest;
+import com.titanic.fork.web.dto.request.account.ValidateNameAndPasswordRequest;
+import io.swagger.annotations.ApiOperation;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.persistence.NoResultException;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("account")
+public class LoginController {
+
+    private final LoginService loginService;
+
+    @ApiOperation(value = "비밀번호 수정을 위한 인증API")
+    @PostMapping("find")
+    public ResponseEntity<Void> validateNameAndPassword(@RequestBody ValidateNameAndPasswordRequest
+                                                                validateNameAndPasswordRequest) {
+        return loginService.validateNameAndPassword(validateNameAndPasswordRequest);
+    }
+
+    @ApiOperation(value = "비밀번호 수정API")
+    @PutMapping("find")
```

Comment:
여기도 Put 메서드행위할 대상을 적어주면 되겠죠?
```suggestion
    @PutMapping("/user")
```

Reply:
지금 보니 account로 하셨군요 account라고 하시면 되겠네요